### PR TITLE
fix: generate test graphs using prop

### PR DESCRIPTION
**Pull request by bryan-terra**
- the test prop wasn't working before
- add the correct param to api call to generate test graph

### CODE REVIEW

**Comment by naheed-shamim** [comment]
I dropped by to check the PR, I observed the test variable is declared but not used.

File: src/graph.tsx [hunk 1]
```diff
@@ -85,9 +85,10 @@ function Graph(props: GraphPropsType) {
     );
   }
   // params of the Graph API render_SDK request
+  const test = props.test ? `&test=${props.test}`: '';
   const getImg = props.getImg ? `&get_img=${props.getImg}` : '';
-  const imgWidth = props.imgWidth ? `&get_img=${props.imgWidth}` : '';
-  const imgHeight = props.imgHeight ? `&get_img=${props.imgHeight}` : '';
+  const imgWidth = props.imgWidth ? `&imgWidth=${props.imgWidth}` : '';
+  const imgHeight = props.imgHeight ? `&imgHeight=${props.imgHeight}` : '';
```
Comment (terraJeffrey) [inline src/graph.tsx:91]: img_width / img_height

File: src/graph.tsx [hunk 2]
```diff
@@ -85,9 +85,10 @@ function Graph(props: GraphPropsType) {
     );
   }
   // params of the Graph API render_SDK request
+  const test = props.test ? `&test=${props.test}`: '';
```
Comment (terraJeffrey) [inline src/graph.tsx:88]: test is not being used

**Review by terraJeffrey — CHANGES_REQUESTED** [review]

File: src/graph.tsx (hunk 2 above)
Comment (bryan-terra) [reply src/graph.tsx:88]: i added it to our backend

Comment (bryan-terra) [reply src/graph.tsx]: nvm i c what u mean

**Review by terraJeffrey — APPROVED** [review]

**Pull request merged**